10a-walkthrough-6.Rmd

```diff
@@ -0,0 +1,677 @@
+
+There are many ways to do this, but we'll choose boxplots to show our audience that there has been roughly twice as many male students in special education than female students between 2012 and 2017. For our purposes, let's verify this by looking at the median male to female ratio for each year: 
+
+```{r}
+model_data %>% 
+    group_by(year) %>% 
+    summarise(median_ratio = median(ratio))
+```
+
+Now let's visualize this using our boxplots: 
+
+```{r visualize female to male ratio}
+model_data %>% 
+    gather(gender, students, c(f, m)) %>% 
+    ggplot(aes(x = year, y = students, color = gender)) + 
+    geom_boxplot() + 
+    scale_y_continuous(labels = scales::comma) + 
+    labs(title = "Median male and female student counts in special education", 
+         subtitle = "Ages 6-21. Does not include outlying areas and freely associated states", 
+         x = "", 
+         y = "", 
+         caption = "Data: US Dept of Education")
+```
+
+```{r visualize female to male ratio 2}
+model_data %>% 
+    gather(gender, students, c(f, m)) %>%
+    ggplot(aes(x = year, y = students)) + 
+    geom_boxplot() + 
+    facet_grid(~ gender) +
+    scale_y_continuous(labels = scales::comma) + 
+    labs(title = "Median male and female student counts in special education", 
+         subtitle = "Ages 6-21. Does not include outlying areas and freely associated states", 
+         x = "", 
+         y = "", 
+         caption = "Data: US Dept of Education")
+```
+
+```{r visualize female to male ratio 3}
+library(ggridges)
+
+model_data %>% 
+    gather(gender, students, c(f, m)) %>%
+    ggplot(aes(x = students, y = gender)) + 
+    geom_density_ridges() +
+    scale_x_continuous(labels = scales::comma) +
+    labs(title = "Density estimates for male and female student counts in special education", 
+         subtitle = "Ages 6-21. Does not include outlying areas and freely associated states", 
+         x = "", 
+         y = "", 
+         caption = "Data: US Dept of Education")
+```
+
+Once we learned from our model that male to female ratios did not change in any meaningful way from 2012 to 2017 and that the median ratio across states was about 2 male students to every female student, we can present these two ideas using this plot. When discussing the plot, it helps to have your model output available to you so you can reference specific actual coefficient estimates when needed. 
```

Comment:
this entire modeling section is amazing - and will be an incredible resource to readers (including myself).

Reply:
thoughts on adding a summary of this section (or perhaps the exploration of all of the data, including processing it far earlier on) here? we cover a lot of group, from exploring the data to modeling it and communicating findings.

Reply:
I'd be totally open to that! In general, I'm cool with taking any of the walkthrough sections and moving it to an earlier chapter for more development